core/tests/protractor_utils/AdminPage.js

```diff
@@ -94,7 +94,8 @@ var AdminPage = function(){
     };
   }
 
-  var saveConfigProperty = function(configProperty) {
```

Comment:
Why change this? I don't see anywhere else in your code that requires it.

Reply:
@brianrodri This was not exactly right. I had to add the params to make it work.